lib/taskana-core/src/main/java/pro/taskana/ClassificationService.java

```diff
@@ -37,12 +43,12 @@
      * domain.
      *
      * @param key
-     *            TODO
+     *            the key of the searched-for classifications
      * @param domain
-     *            TODO
+     *            the domain of the searched-for classifications
      * @return If exist: domain-specific classification, else root classification
      * @throws ClassificationNotFoundException
-     *             TODO
+     *             if no classification is found that matches key and domain.
```

Comment:
Es wird die classification der root-Domäne geliefert, sofern diese existiert. Folglich sollte die Exception nur kommen, wenn der Key unbekannt ist und nicht einmal in der root-Domäne existiert.

Reply:
Geändert